fabric/src/main/resources/data/scannable/recipes/chest_module.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "minecraft:crafting_shapeless",
+    "category": "misc",
+    "group": "scanner_module",
+    "ingredients": [
+        {
+            "item": "scannable:blank_module"
+        },
+        {
+            "item": "minecraft:chest"
+        }
+    ],
+    "result": {
+        "item": "scannable:chest_module"
+    }
+}
```

Comment:
Please stick to formatting of other data files for consistency (using 2 space indentation, which IIRC is also what pops out when running the data generators).